docs/getting-started.md

```diff
@@ -164,8 +164,17 @@ You should see something like this:
 ↳ <u><a href="http://127.0.0.1:3000/" style="color: inherit;">http://127.0.0.1:3000/</a></u></pre>
 
 <div class="note">
-  <p>If port 3000 is in use, the preview server will choose the next available port, so your actual port may vary. To specify port 4321 (and similarly for any other port), use <nobr><code>--port 4321</code></nobr>.</p>
-  <p>For security, the preview server is by default only accessible on your local machine using the <a href="https://en.wikipedia.org/wiki/Localhost">loopback address</a> 127.0.0.1. To allow remote connections, use <nobr><code>--host 0.0.0.0</code></nobr>.</p>
+
+If port 3000 is in use, the preview server will choose the next available port, so your actual port may vary. To specify a different port, use <nobr>`--port`</nobr>. For example, to run the preview server with npm on port 4321:
+
+> `npm run dev -- --port 4321`
+
+For security, the preview server is by default only accessible on your local machine using the [loopback address](https://en.wikipedia.org/wiki/Localhost) 127.0.0.1. To allow remote connections, use <nobr>`--host 0.0.0.0`</nobr>.
+
+Port numbers below 1024 are usually protected and might need admin priviledges. For example, to share your preview server on the default http port:
+
+> `sudo yarn dev --host 0.0.0.0 --port 80`
```

Comment:
```suggestion
> `sudo npm run dev -- --host 0.0.0.0 --port 80`
```

I think we should bias towards npm rather than Yarn and be consistent if we’re only showing one.